component/src/main/java/org/cbioportal/genome_nexus/component/annotation/CanonicalTranscriptResolver.java

```diff
@@ -52,4 +66,45 @@ else if (transcripts.size() > 1) {
                 variantAnnotation.getTranscriptConsequences(), variantAnnotation.getMostSevereConsequence());
         }
     }
+
+    /**
+     * Relies on VEP canonical assignment.
+     */
+    public List<TranscriptConsequence> findCanonicalTranscriptCandidates(
```

Comment:
@onursumer wondering if this function should maybe be private just to avoid people from accidentally using it?